src/router/index.ts

```diff
@@ -36,17 +37,19 @@ const routes: Array<RouteConfig> = [
     path: "/alerts",
     name: "alerts",
     component: LrAlert,
-    props:true
+    props: true,
+  },
+  {
+    path: "/manage",
+    name: "manage",
+    component: LrManageUser,
+    props: true,
```

Comment:
You don't need this flag if you are not passing any prop in route url

Reply:
The flag has been removed.